indico/modules/users/tasks.py

```diff
@@ -68,9 +68,14 @@ def export_user_data(user, options):
     if export_request and export_request.is_running:
         return
 
-    # ughh why isn't there an ORM upsert..
+    # If an export request already exists we delete it
+    # along with the actual file (if any) and create a new request
     if export_request:
+        f = export_request.file
         db.session.delete(export_request)
+        if f:
```

Comment:
I'd just set `f.claimed = False` - that way the task deals with it which has much nicer error handling (see `delete_unclaimed_files`)